dask/dataframe/methods.py

```diff
@@ -276,12 +276,14 @@ def unique(x, series_name=None):
     return out
 
 
-def value_counts_combine(x):
-    return x.groupby(level=0).sum()
+def value_counts_combine(x, dropna=True):
+    return x.groupby(level=0, dropna=dropna).sum()
 
 
-def value_counts_aggregate(x):
-    return x.groupby(level=0).sum().sort_values(ascending=False)
+def value_counts_aggregate(x, sort=True, ascending=False, dropna=True):
+    if sort:
+        return x.groupby(level=0, dropna=dropna).sum().sort_values(ascending=ascending)
+    return x.groupby(level=0, dropna=dropna).sum()
```

Comment:
To avoid duplication maybe we can bring the `x.groupby(level=0, dropna=dropna).sum()` bit outside of the conditional, nd then apply the `sort_values` method call on that if there is a sort.

Reply:
:+1: I'll change it.